docker-compose.yml

```
@@ -1,4 +1,4 @@
-version: '2'
+version: '3.7'
```

This is a docker-compose format version, not python. Need to revert that back to '2'
```suggestion
version: '2'
```